shiftcommerce/shift-react-components
# Trendy Issue/111: Checkout Place Order Updates

## Overview

Github Issue: https://github.com/shiftcommerce/trendy-golf/issues/111

This PR removes the delete line item link in the checkout cart and adds logic to render a credit card icon for card payments.

It also bumps the package version to `v0.12.5`

## QA

1. checkout Shift Frontend React branch `trendy_golf_issue/111_checkout_updates`

2. Add items to cart and navigate to the first step of checkout - `payment-method` selection

3. Select `Pay By Credit/Debit Card` option and ensure the payment methods summary component is displayed on `Step 2 - Shipping Address`

![Payment Methods Summary for Card](https://user-images.githubusercontent.com/4486874/57382299-97640300-71a4-11e9-8634-59ecce0a06b8.png)

4.  Navigate to the checkout review and submit page. Check that the `Place Order` buttons are the same colour

![Checkout 5](https://user-images.githubusercontent.com/4486874/57376699-99c06000-7198-11e9-9d0c-64741f318091.png)

5. Also check that the delete link is not present for cart line items in checkout
